docs/sdks/js-ts/migration.mdx

```diff
@@ -205,7 +216,7 @@ const res = await kwil.call(msg);
 */
 ```
 
-If a `view` action has a `must_sign` auxiliary, you should also chain `.signer()` and `.publicKey()` methods to the builder. You can also chain a `.description()` to customize the signature message.
+If a `view` action has a `mustsign` auxiliary, you can also pass a `KwilSigner` as a second argument to the `kwil.call()` method.
```

Comment:
`mustsign` will be removed before the release

Reply:
I will leave this in the docs for now and then remove once `mustsign` is removed from kwil-db.